StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/PaneLayoutView/PaneLayoutView+Extensions.swift

```diff
@@ -0,0 +1,213 @@
+//
+//  PaneLayoutView+Header.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 12/22/23.
+//
+
+import Foundation
+@_spi(STP) import StripeUICore
+import SwiftUI
+import UIKit
+
+extension PaneLayoutView {
+
+    @available(iOSApplicationExtension, unavailable)
+    static func createContentView(
+        iconView: RoundedIconView?,
+        title: String?,
+        subtitle: String?,
+        contentView: UIView?,
+        isSheet: Bool = false
+    ) -> UIView {
+        let verticalStackView = UIStackView()
+        verticalStackView.axis = .vertical
+        verticalStackView.spacing = 0
+        if iconView != nil || title != nil {
+            let headerView = createHeaderView(
+                iconView: iconView,
+                title: title,
+                isSheet: isSheet
+            )
+            verticalStackView.addArrangedSubview(headerView)
+        }
+        if subtitle != nil || contentView != nil {
+            let bodyView = createBodyView(
+                text: subtitle,
+                contentView: contentView
+            )
+            verticalStackView.addArrangedSubview(bodyView)
+        }
+        return verticalStackView
+    }
+
+    @available(iOSApplicationExtension, unavailable)
+    static func createHeaderView(
+        iconView: RoundedIconView?,
+        title: String?,
+        isSheet: Bool = false
+    ) -> UIView {
+        let headerStackView = HitTestStackView()
+        headerStackView.axis = .vertical
+        headerStackView.spacing = 16
+        headerStackView.alignment = .leading
+        if let iconView = iconView {
+            headerStackView.addArrangedSubview(iconView)
+        }
+
+        if let title = title {
+            let titleLabel = AttributedTextView(
+                font: .heading(.large),
+                boldFont: .heading(.large),
+                linkFont: .heading(.large),
+                textColor: .textDefault
+            )
+            titleLabel.setText(title)
+            headerStackView.addArrangedSubview(titleLabel)
+        }
+
+        let paddingStackView = HitTestStackView(
+            arrangedSubviews: [
+                headerStackView
+            ]
+        )
+        paddingStackView.isLayoutMarginsRelativeArrangement = true
+        paddingStackView.directionalLayoutMargins = NSDirectionalEdgeInsets(
+            top: isSheet ? 0 : 16, // the sheet handle adds some padding
```

Comment:
nit: move the doc for `isSheet` to `createContentView` for better readability

Reply:
I could be misunderstanding, but the comment here, while far from perfect, is trying to say "the 0 for isSheet is acceptable because the sheet context will add extra padding elsewhere" - from that sense it is trying to add the comment right where its needed: by the 0 vs. 16 padding

the lack of (or need) for `isSheet` documentation is a different (but similar) concern